pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -353,6 +353,10 @@ func (s *nodeTopoInformer) reportNodeTopology() {
 		klog.V(5).Infof("feature %v not enabled, node topo will not be reported", features.NodeTopologyReport)
 	}
 
+	if !koordletutil.NodeCPUInfoInited {
+		return
+	}
+
 	nodeTopoAnnotations, err := s.calcNodeTopo()
```

Comment:
how about we just ignore the not exist error from metric cache, so that we don't need to add an atomic flag in util(actually that's not the duty of util pkg).

Reply:
well